app/views/api/v2/smart_proxies/show.json.rabl

```diff
@@ -11,6 +11,7 @@ if @version
 end
 
 child :smart_proxy_features => :features do
+  attributes :capabilities, :settings
```

Comment:
I wonder if this also should be added to the index at https://github.com/theforeman/foreman/blob/2e400f66c41b6883d6ab1e4ca1de71b84618890a/app/views/api/v2/smart_proxies/main.json.rabl#L8

Reply:
Settings can be private and contain sensitive things. I'm not sure we should expose all of them. Capabilities should be safe since they're a list of constants.

Reply:
I don't think that *exposed* settings should have sensitive things in them. They're *exposed* for a reason? ;-)